### ISSUE TITLE
vcf first 8 columns without genotype columns

### ISSUE THREAD

**Issue by zhouhufeng** [samtools/bcftools#896]
I am wondering if it is possible for bcftools to work on the vcf file with only first 8 columns without genotype columns? I just need to break the multi-allelic vcf into separate rows. But bcftools can't recoginze the vcf format (only first 8 columns)

**Comment by pd3** [comment]
It should be possible. Can you show which command fails and how you run it, you are not giving enough details.

**Comment by zhouhufeng** [comment]
> It should be possible. Can you show which command fails and how you run it, you are not giving enough details.

Thanks very much, here attached some examples: 

`bcftools norm -m -any human.hg38.vcf -Ov -o human.hg38.multiple.vcf`
[E::vcf_hdr_read] No sample line
Failed to open human.hg38.vcf: could not parse header

**Comment by pd3** [comment]
It seems you are working with an invalid VCF. The line starting with `#CHROM POS ...` must be always present. Please consult the VCF specification http://samtools.github.io/hts-specs/VCFv4.3.pdf.

**Comment by zhouhufeng** [comment]
> It seems you are working with an invalid VCF. The line starting with `#CHROM POS ...` must be always present. Please consult the VCF specification http://samtools.github.io/hts-specs/VCFv4.3.pdf.

Yeah, this is actually a GNOMAD VCF downloaded from GNOMAD official website. The header seems to be correct. 

##reference=file:///seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
3	131700921	.	AG	A	274.67	PASS	AB_HIST_ALL=0|0|0|0|0|0|0|0|0|0|0|0|1|0|0|0|0|0|0|0;AB_HIST_ALT=0|0|0|0|0|0|0|0|0|0|0|0|1|0|0|0|0|0|0|0;AB_MEDIAN=6.15385e-01;AC=1;AC_AFR=0;AC_AMR=0;AC_ASJ=0;AC_EAS=0;AC_FIN=
0;AC_Female=1;AC_Male=0;AC_NFE=0;AC_OTH=1;AC_POPMAX=.;AC_raw=1;AF=3.52759e-05;AF_AFR=0.00000e+00;AF_AMR=0.00000e+00;AF_ASJ=0.00000e+00;AF_EAS=0.00000e+00;AF_FIN=0.00000e+00;AF_Female=7.86287e-05;AF_Male=0.00000e+00;AF_NFE=0.00000e+00;AF_O
TH=1.06383e-03;AF_POPMAX=.;AF_raw=3.37769e-05;AN=28348;AN_AFR=6886;AN_AMR=700;AN_ASJ=294;AN_EAS=1508;AN_FIN=3438;AN_Female=12718;AN_Male=15630;AN_NFE=14582;AN_OTH=940;AN_POPMAX=.;AN_raw=29606;AS_FilterStatus=PASS;AS_RF=6.45339e-01;BaseQRa
nkSum=1.38000e+00;CSQ=-|intergenic_variant|MODIFIER||||||||||||||||1||||deletion|1||||||||||||||||||||||||||||||||||||||||||||;ClippingRankSum=8.17000e-01;DP=525804;DP_HIST_ALL=128|477|1461|1945|2207|2247|2764|2978|540|44|10|1|0|0|0|1|0|0
|0|0;DP_HIST_ALT=0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0;DP_MEDIAN=13;DREF_MEDIAN=1.99526e-34;FS=0.00000e+00;GC=14173,1,0;GC_AFR=3443,0,0;GC_AMR=350,0,0;GC_ASJ=147,0,0;GC_EAS=754,0,0;GC_FIN=1719,0,0;GC_Female=6358,1,0;GC_Male=7815,0,0;GC_
NFE=7291,0,0;GC_OTH=469,1,0;GC_raw=14802,1,0;GQ_HIST_ALL=87|42|26|115|249|252|578|732|395|864|875|448|1305|394|952|511|1313|235|1308|4122;GQ_HIST_ALT=0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|1;GQ_MEDIAN=99;Hom=0;Hom_AFR=0;Hom_AMR=0;Hom_ASJ=0
;Hom_EAS=0;Hom_FIN=0;Hom_Female=0;Hom_Male=0;Hom_NFE=0;Hom_OTH=0;Hom_raw=0;InbreedingCoeff=3.90000e-03;MQ=5.69600e+01;MQRankSum=-1.38100e+00;OriginalContig=10;OriginalStart=47993347;POPMAX=.;QD=2.11300e+01;ReadPosRankSum=-1.09200e+00;Reve
rseComplementedAlleles;SOR=3.07000e-01;VQSLOD=-2.33700e+00;VQSR_NEGATIVE_TRAIN_SITE;VQSR_culprit=SOR

**Comment by pd3** [comment]
The header does not seem complete. Try to run `vcf-validator` from the vcftools package to make sure your VCF is valid, from your description it seems not.
